src/friendly-errors-plugin.js

```diff
@@ -98,6 +99,31 @@ class FriendlyErrorsWebpackPlugin {
   }
 }
 
+function extractErrorsFromStats(stats, type) {
+  if (isMultiStats(stats)) {
+    const errors = stats.stats
+      .reduce((errors, stats) => errors.concat(extractErrorsFromStats(stats, type)), []);
+    // Dedupe to avoid showing the same error many times when multiple
+    // compilers depend on the same module.
+    return uniqby(errors, error => error.message);
```

Comment:
What about:

```js
function uniqueBy(arr, fun) {
  const seen = {};
  return arr.filter(el => {
    const e = fun(el);
    return !(e in seen) && (seen[e] = 1);
  })
}
```

to avoid lodash dependency ?